<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEventDialog</class>
 <widget class="QDialog" name="AddEventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加日程事件</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>事件名称:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="locationLabel">
       <property name="text">
        <string>地点:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="locationEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>描述:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="descriptionEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="startTimeLabel">
       <property name="text">
        <string>开始时间:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateTimeEdit" name="startTimeEdit">
       <property name="displayFormat">
        <string>yyyy-MM-dd hh:mm</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="endTimeLabel">
       <property name="text">
        <string>结束时间:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateTimeEdit" name="endTimeEdit">
       <property name="displayFormat">
        <string>yyyy-MM-dd hh:mm</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="isCourseLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="isCourseCheck">
       <property name="text">
        <string>是否为课程/办公时间</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>AddEventDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>AddEventDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
